### PR TITLE
Update taint-and-toleration.md

### PR DESCRIPTION
**Pull request by dalthonmh**
Command is added to get all nodes with the associated taints

### CODE REVIEW

**Comment by dipesh-rawat** [comment]
@dalthonmh Thanks you for your contribution.
Could you please sign the CLA before the PR can be reviewed.
You can follow the steps documented here: [CLA Steps](https://git.k8s.io/community/CLA.md)

**Review by kundan2707 — COMMENTED** [review]
Addition is not at correct place.
It should come after basic addition removing, basic listing.
Just addition at first command does not seems suitable

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ marks that the node should not accept any pods that do not tolerate the taints.
 
 ## Concepts
 
+To get the list of nodes with the associated taints use the command
+```shell
+kubectl get nodes -o custom-columns=NODE:.metadata.name,TAINTS:.spec.taints
+```
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:34]: This isn't the right place for this advice, I'm afraid.

**Review by sftim — COMMENTED** [review]
I don't think we should make this change @dalthonmh 

You're welcome to make a case for why we should, though.

/hold
Let's not merge this until we've been convinced otherwise.

**Pull request closed without merging**